### PR TITLE
feat: Add ros2_control doxygen documentation support

### PR DESCRIPTION
**Pull request by jaron-l**
* Add GitHub workflow step to checkout ros2_control
* Add breathe settings to conf.py

This is in reference to [ros2_control #332](https://github.com/ros-controls/ros2_control/issues/332)

### CODE REVIEW

File: .github/workflows/sphinx-make-page.yml [hunk 1]
```diff
@@ -11,6 +11,11 @@ jobs:
   build-deploy:
     runs-on: ubuntu-latest
     steps:
+    - name: checkout ros2_control
+      uses: actions/checkout@v2
+      with:
+        repository: ros-controls/ros2_control
+        path: ros2_control
```
[the remark this answers is not in the document]
Comment (jaron-l) [reply .github/workflows/sphinx-make-page.yml:18]: I briefly looked into it. I don't think we would even need the deployment scripts repo. You can already do everything it does with the modular approach of combining already existing GitHub actions.

**Comment by bmagyar** [comment]
Yes guys please fix it in whatever way we can :D I personally prefer the deployment script as that allows (or at least used to) local testing and keeps all the deployment logic in one place. *However* I had no time for fixing it and not likely I will have in the near future.

**Comment by jaron-l** [comment]
@destogl and @bmagyar, my recent commits show the fully converted workflows to GitHub actions. I added documentation on how you can locally test the workflows to the README.md to address @bmagyar's concern. I still think this way is easier as it requires less code maintenance from us for any future changes (though that does require using an external package to make it really easy). Let me know what you think.

**Comment by bmagyar** [comment]
Thanks very much for the effort, especially for the readme ;)

**Pull request merged**